config.py

```diff
@@ -79,6 +82,30 @@ def __init__(self):
         execution engine. Must be supplied.
         """
 
+        # ==========================================================================================
+        # Geth Configuration
+
+        self.data_dir = "db/L2-execution"
+        """Geth data directory for op-geth node."""
+
+        self.chaindata_dir = f"{self.data_dir}/geth/chaindata"
+        """Directory storing chain data."""
+
+        self.chain_id = 42069
```

Comment:
I set this to be the chain_id in the getting-started docs, lmk if there should be a different default

Reply:
I think down the line we want some parameters to be mandatory and this should probably be one of them. For now this is fine!